source/sass/wagtail/profiles.scss

```diff
@@ -1,89 +1,78 @@
-.profiles {
-  .headshot {
-    width: 100%;
-    max-width: 160px;
-    max-height: 160px;
-    object-fit: cover;
-    object-position: top;
-
-    &-container {
-      //Mobile & Desktop
-
-      @media screen and (min-width: $bp-sm) {
-        grid-row-end: 3;
-        grid-row-start: 1;
-      }
-
-      //Tablet
-
-      @media screen and (min-width: $bp-md) and (max-width: $bp-lg) {
-        grid-row-start: 1;
-        grid-row-end: 2;
-      }
-    }
-  }
-}
-
 .person-card {
-  //Mobile & Desktop
+  $headshot-width: 92px;
+  $headshot-height: 92px;
+  $headshot-width-lg: 160px;
+  $headshot-height-lg: 160px;
+  $gutter: 14px;
+  $card-padding-top: 10px;
 
   border-top: 1px solid $black;
-  display: grid;
-  grid-template-rows: repeat(2, 0.25fr);
-  grid-template-columns: 1fr 2fr;
 
-  @media screen and (min-width: $bp-sm) {
-    grid-template-rows: 0.5fr 0.75fr;
-    grid-template-columns: 0.9fr 2fr;
-    min-height: 160px;
+  @media screen and (min-width: $bp-lg) {
+    position: relative;
+    min-height: $headshot-height-lg;
   }
 
-  //Tablet
+  .thumbnail-wrapper {
+    float: left;
 
-  @media screen and (min-width: $bp-md) and (max-width: $bp-lg) {
-    grid-template-rows: repeat(2, 0.25fr);
-    grid-template-columns: 1fr 2fr;
-    min-height: 160px;
+    @media screen and (min-width: $bp-lg) {
+      float: none;
```

Comment:
Nitpick: Does `float` actually do anything if the element is absolutely positioned?

From https://developer.mozilla.org/en-US/docs/Web/CSS/float
>The float CSS property places an element on the left or right side of its container, allowing text and inline elements to wrap around it. The element is removed from the normal flow of the page, though still remaining a part of the flow (in contrast to absolute positioning).

Reply:
yea probably! i dont know what i had that in the first place lol

Reply:
Yea probably! I dont know why I had that in the first place lol maybe just to keep mobile/desktop rule consistent 🤷‍♀ 

I'll test it again if if that `float: none` doens't do anything I'll remove that line.

Reply:
To make sure we're on the same page: I think you can remove the `float:none` because `position:absolute` makes the float not do anything.

Reply:
Removed.